aten/src/ATen/native/vulkan/Vulkan.cpp

```diff
@@ -1025,12 +1025,17 @@ class VulkanTensor::Impl {
         initVulkanContextOnce(), "Vulkan Failed to create Vulkan Context");
     numel_ = std::accumulate(
         std::begin(sizes_), std::end(sizes_), 1, std::multiplies<int64_t>());
+    strides_ = std::vector<int64_t>(sizes_.size());
```

Comment:
You can consider initializing this in the constructor to avoid a default initialization only to be overwritten later.